tools/run-e2e-test.ps1

```diff
@@ -0,0 +1,66 @@
+### This script is a quick way to run the XHarness E2E tests.
+### These test are located in tests/integration-tests and require the Arcade and Helix SDK.
+### To run them, you need to invoke these through MSBuild which makes the process a bit cumbersome.
+### This script should make things easier.
+###
+### Usage: .\run-e2e-test.ps1 -TestProject Apple/SimulatorInstaller.Tests.proj [-SkipBuild]
+
+param (
+    [Parameter(Mandatory = $true)]
+    [string]
+    $TestProject,
+    
+    [switch]
+    $SkipBuild = $false
+)
+
+$repoRoot = "$PSScriptRoot\.."
+
+function Write-Projects {
+    Write-Output "Possible options:"
+    $files = Get-ChildItem -Include *Tests.proj -Recurse -Path "$repoRoot\tests\integration-tests\Android"
+    foreach ($item in $files) {
+        '  - Android\{0}' -f $item.Name
+    }
+
+    $files = Get-ChildItem -Include *Tests.proj -Recurse -Path "$repoRoot\tests\integration-tests\Apple"
+    foreach ($item in $files) {
+        '  - Apple\{0}' -f $item.Name
+    }
+
+    $files = Get-ChildItem -Include *Tests.proj -Recurse -Path "$repoRoot\tests\integration-tests\WASM"
+    foreach ($item in $files) {
+        '  - WASM\{0}' -f $item.Name
+    }
+}
+
+if (-not(Test-Path -Path $TestProject -PathType Leaf)) {
+    $TestProject = "$repoRoot\tests\integration-tests\$TestProject"
```

Comment:
If this script is designed to be called manually then I think it would be more convenient if `$TestProject` was relative to current directory instead of `intergration-tests` folder because then I could use tab completion to fill the path instead of having to type or paste it manually.

Reply:
That works too, just if it doesn't find it, it tries the other version (relative to integration-tests):
```
λ  .\tools\run-e2e-test.ps1 .\tests\integration-tests\Apple\Simulator.Commands.Tests.proj
```


I kept this behavior same as in the shell version of the script.